src/ql/pass/map/qubits/map/map.cc

```diff
@@ -403,11 +351,8 @@ pmgr::pass_types::NodeType MapQubitsPass::on_construct(
     // Build the options structure for the mapper.
     parsed_options.emplace();
 
-    parsed_options->initialize_one_to_one = options["initialize_one_to_one"].as_bool();
```

Comment:
I have updated the tests to not set the option anymore.

This didn't make any fail test, which is surprising. I expect the opposite, if the option was well tested and actually did something.

I will try to investigate more. But:
- the unit test I added should be IMO the new way forward to test a single pass or class. Integration tests using the Python API should be in small numbers, testing global options and overall integration. C++ unit tests are wayyy better for debugging, and they allow better code coverage and it's easier to reason at the class level, rather than at the whole program level.
- I noticed that a lot of those Python tests only test that the cqasm input compiles, they do not check that the actual output corresponds to a correct output (except those using the "golden" files), which is not strong
- There seems to be some good cleanup to do in the tests folder.